### DISCUSSION TITLE
Win7x64 | Osu lazer crash

### DISCUSSION THREAD

**Discussion by andrey6498** [ppy/osu#15477] [Q&A] [answered]
Probably lazer is just unsupported on win 7 but it was running properly on older lazer releases(quite old ones though)
___
[runtime.log](https://github.com/ppy/osu/files/7478342/runtime.log)
[DxDiag.txt](https://github.com/ppy/osu/files/7478344/DxDiag.txt)

**Comment by bdach** [comment, marked as answer]
correct, windows 7 is no longer supported.

**Reply by andrey6498** [reply]
so realm framework can't run properly on win 7 i guess? Or there's some other cause(seems like it can't find entry point or something)? Dlls are 100% sure present in folder

**Reply by bdach** [reply]
correct. realm does not support windows 7 and is unlikely to add support in the future.
